docs/data/material/guides/understand-mui-packages/understand-mui-packages.md

```diff
@@ -2,106 +2,62 @@
 
-This layer is specifically related to stylesheet generation (CSS-in-JS).
-MUI has introduced new styled-engines in v5 to unlock more possibilities and enable enhanced customization.
-Usually, developers do not need to interact with this layer on a daily basis because it is used internally in `@mui/system`.
+Each of these opportunities are now individual packages within our two main product lines, MUI Core and MUI X:
 
-They come in two packages:
+<img src="/static/images/packages/mui-packages.png" style="width: 796px; margin-top: 4px; margin-bottom: 8px;" alt="The first half of the image shows @mui/material and @mui/base as component libraries, and @mui/system and styled engines as styling solutions, both under the MUI Core umbrella. The second half shows @mui/x-data-grid and @mui/x-date-pickers as components from MUI X."/>
 
-- `@mui/styled-engine` : an [emotion](https://emotion.sh/docs/styled) wrapper.
-- `@mui/styled-engine-sc` : a [styled-components](https://styled-components.com/docs/basics#getting-started) wrapper.
+In this article, we'll only cover the MUI Core packages.
+Check the [MUI X overview](/x/advanced-components) for more information about our collection of advanced components.
 
-These adapters unify the APIs of both `emotion` and `styled-components`, so that developers can choose to use whichever suits them best.
+## Component libraries
 
-The previous style library `@mui/styles` (JSS wrapper) is deprecated and will be removed in the future.
-
-> The details about the change in styling solution are in [this RFC](https://github.com/mui/material-ui/issues/22342).
-
-## System
-
-There is only one package in this layer - `@mui/system`.
-
-It uses the emotion adapter (`@mui/styled-engine`) as the default styled-engine to create APIs for building a design system from scratch. For example, [`styled`](/system/styled/#main-content) from the styled-engine is enhanced to provide more theming capabilities.
-
-If you want to switch the styled-engine to use styled-components, [follow this guide](/material-ui/guides/styled-engine/#how-to-switch-to-styled-components).
-
-<img src="/static/images/packages/mui-system.png" style="display: block; width: 720px; margin: 40px auto;" />
-
-Here are some benefits:
-
-- You have full control of the `theme` object.
-- The `styled` API supports the `sx` prop by default.
-- Components created with `styled` are themeable via slots & variants.
+### Material UI
 
-> **Note**: you will have to install either `emotion` or `styled-components`, because the respective styled-engine package depends on it.
+Material UI is a comprehensive library of components that features our implementation of Google's Material Design system. It has `@mui/system` and `@mui/base` as dependencies, meaning you don't need to install or import them seperately—importing any modules you need from `@mui/material` itself will work.
```

Comment:
I thought the first snippet was possible as well — cc @michaldudak for further disambiguation!

Reply:
@danilo-leal the first way used to be possible, but we removed this possibility some time ago, so that our products have more clearly defined boundaries. We no longer export Base components from Material UI, so only `import SwitchUnstyled from '@mui/base/SwitchUnstyled';` is valid.

Reply:
So, removing the part where we say that you can import Base components from @mui/material would be the correct thing to do? (MUI System still applies though, I suppose)

Reply:
Yes, that's right.